packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -2340,19 +2757,20 @@ describe('ReactErrorBoundaries', () => {
       return <Throws />;
     }
 
-    ReactDOM.render(
-      <ErrorBoundary>
-        <Wrapper />
-      </ErrorBoundary>,
-      container,
-    );
+    await act(async () => {
+      root.render(
```

Comment:
This error is not caught by the error boundary and throws and causes this test to fail.